Lib/_dummy_thread.py

```diff
@@ -71,6 +71,10 @@ def get_ident():
     """
     return 1
 
+def get_native_id():
+    """Dummy implementation of _thread.get_native_id()."""
+    return None
```

Comment:
None is not a non-negative integer. It contradicts the doc, no?

Reply:
The doc will be updated – this change reflects your suggestion from the previous PR (to have the value be `None` if not started, matching the behavior of`ident`)

Reply:
@vstinner should `dummy_thread` be updated to remove the function if the OS doesn't support the capability, or does it not matter since it always returns `None`?